src/tools/curve_format_main.cpp

```diff
@@ -86,11 +107,119 @@ DEFINE_bool(needWriteZero,
         true,
         "not write zero for test.");
 
-using curve::fs::FileSystemType;
-using curve::fs::LocalFsFactory;
-using curve::fs::FileSystemInfo;
-using curve::fs::LocalFileSystem;
-using curve::common::kFilePoolMaigic;
+DEFINE_string(fileSystemType,
+    "ext4",
+    "file system type");
+
+DEFINE_string(pfs_cluster,
+    "spdk",
+    "pfs cluster name, only for pfs");
+
+DEFINE_string(pfs_pbd_name,
+    "0000:c2:00.0n1",
+    "pfs pbd name, only for pfs");
+
+DEFINE_int32(pfs_host_id,
+    2,
+    "pfs host id, only for pfs");
+
+#ifdef WITH_SPDK
+void
+glog_pfs_func(int level, const char *file, const char *func, int line,
```

Comment:
This part of the function is also defined in chunkserver.cpp.

Reply:
done